docs/training-apis/deepspeed/pytorch2deepspeed.rst

```diff
@@ -0,0 +1,79 @@
+.. _pytorch-to-deepspeed:
+
+####################################################
+`PyTorchTrial` to `DeepSpeedTrial`
+####################################################
+
+Adapting an existing :class:`~determined.pytorch.PyTorchTrial` to use DeepSpeed is
```

Comment:
Change to: "Adapting an existing :class:`~determined.pytorch.PyTorchTrial` to use DeepSpeed mirrors the process for adapting existing code to use DeepSpeed outside of Determined."
This is not a marketing doc.